tests/test_window.py

```diff
@@ -9,12 +9,12 @@
 class WindowTest(WindowTestCase):
 
     def test_window_title(self):
-        """Tests the window title"""
+        """Tests the window's title"""
```

Comment:
Yours is better than mine (i.e., the apostrophe is correct). But in this kind of cases we do not need to follow religiously the english vocabulary, although there's no harm if we do it! 👍